gax-grpc/src/main/java/com/google/api/gax/grpc/AttemptCallable.java

```diff
@@ -78,7 +77,6 @@ public ResponseT call() {
       externalFuture.setAttemptFuture(internalFuture);
     } catch (Throwable e) {
       externalFuture.setAttemptFuture(ApiFutures.<ResponseT>immediateFailedFuture(e));
-      throw e;
```

Comment:
Inspired by the recent "learning on the loo" post, suggesting that same exception should not be logged twice. Setting catched exception into externalFuture will record that exception in another place already, so throwing it from here will duplicate same exception.